### PR TITLE
Update i18n files, make sure we can replace text when necessary, fix styles

### PR DESCRIPTION
**Pull request by vforge**

### CODE REVIEW

File: src/ContentManager.js [hunk 1]
```diff
@@ -26,10 +54,25 @@ export const langToContent = {
...
+                ...content,
+                ...langToContent[lang],
+            };
+        }
+
+        this.content = content;
+    }
+
+    t(key) {
+
```
Comment (JoshuaRogan) [inline src/ContentManager.js]: Is this the translate function that returns a string?

Comment (vforge) [reply src/ContentManager.js]: Oh, this should be removed, one sec

**Comment by JoshuaRogan** [comment]
LGTM Tested on chrome

**Comment by vforge** [comment]
@JoshuaRogan Again, I'll fix tests

File: src/components/App.js [hunk 2]
```diff
@@ -88,7 +88,7 @@ class App extends Component {
                 <ScreenOne
                     appOptions={options}
                     content={content}
-                    text={content.bodyParagraphs}
+                    text={content.mainParagraphs}
```
Comment (mklucsarits) [inline src/components/App.js]: If `content` is already passed in to this component, I think we should just use it directly and remove this `text` prop.

Comment (vforge) [reply src/components/App.js]: thanks for catching it, I already removed this prop in the component

File: src/ContentManager.js [hunk 3]
```diff
@@ -13,7 +13,35 @@ import ru from './i18n/ru/tracking-opt-in.json';
...
 
-export const langToContent = {
+/**
+ * Additional strings (URLs, mostly)
+ * - Keys for this object are the same as keys for `langToContent`
+ * - default values are specified in `en` key
+ */
+const additionalStrings = {
+    en: {
+        privacyPolicyUrl: "https://www.fandom.com/privacy-policy",
```
Comment (mklucsarits) [inline src/ContentManager.js]: Not a blocker but we should use single-quote strings.

Comment (vforge) [reply src/ContentManager.js]: doh, right

**Pull request merged**